fly.toml

```diff
@@ -0,0 +1,43 @@
+# fly.toml file generated for webstudio-dev on 2022-05-21T21:31:32+02:00
+
+app = "webstudio-dev"
```

Comment:
just realizing that this is basically fixed, so we won't be able to have webstudio-prod or webstudio-staging ... how do we go about that?

Reply:
You can pass an `--app` flag to `fly deploy`

https://fly.io/docs/flyctl/deploy/

Renamed it to webstudio aswell

Reply:
but how would you create staging and dev apps? Do we need a different app name? Should it become an env variable?